src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/ConfigurationLiveTests.cs

```diff
@@ -84,13 +87,14 @@ public async Task DeleteSettingNotFound()
         public async Task DeleteSetting()
         {
             ConfigurationClient service = TestEnvironment.GetClient();
+            ConfigurationSetting testSetting = CreateSetting();
 
             try
             {
                 // Prepare environment
-                var testSettingDiff = s_testSetting.Clone();
+                var testSettingDiff = testSetting.Clone();
                 testSettingDiff.Label = null;
-                await service.SetAsync(s_testSetting);
+                await service.SetAsync(testSetting);
```

Comment:
Why are we setting two keys in this test (and next one too)?

Reply:
You are right. We don't need two. Removing...